### DISCUSSION TITLE
Problem compiling Slint image-url with https source.  Works fine in SlintPad

### DISCUSSION THREAD

**Discussion by richardmcatlin** [slint-ui/slint#4131] [Q&A]
This works in SlintPad, but does not compile for cargo project.
```
export component Example inherits Window {
    in property <image> url1: @image-url("https://upload.wikimedia.org/wikipedia/commons/4/42/Berlin_potsdamer_platz.jpg");
    width: 200px;
    height: 150px;
    VerticalLayout {
        Image {
            source: url1;
            width: 200px;
            // implicit default, preserving aspect ratio:
            // height: self.width * natural_height / natural_width;
        }
    }
}
```
error: Cannot find image file https://upload.wikimedia.org/wikipedia/commons/4/42/Berlin_potsdamer_platz.jpg
--> /Users/rhettcatlin/Desktop/catlin/rust/rust_fullstack_on_aws_serverless/client_slint_puzzle/slide_puzzle.slint:112:31
|
112 | in property url1: @image-url("https://upload.wikimedia.org/wikipedia/commons/4/42/Berlin_potsdamer_platz.jpg");

**Comment by tronical** [comment]
Thank you for reporting the issue. It was also reported as https://github.com/slint-ui/slint/issues/4132 and is a duplicate of https://github.com/slint-ui/slint/issues/3279 . At the moment https url support is not implemented in the Slint run-time library. This is a feature we would like to eventually support, but at the moment you'd have to do this yourself in native code (using reqwest in Rust or libcurl in C++ for example).
